protocol/src/main/java/net/md_5/bungee/protocol/LegacyDecoder.java

```diff
@@ -14,13 +14,6 @@
     @Override
     protected void decode(ChannelHandlerContext ctx, ByteBuf in, List<Object> out) throws Exception
     {
-        // See check in Varint21FrameDecoder for more details
-        if ( !ctx.channel().isActive() )
-        {
-            in.skipBytes( in.readableBytes() );
-            return;
-        }
-
```

Comment:
Why remove this?

Reply:
in the description of the pr
we don't need it anymore

Reply:
I don't think it hurts to leave it in as a safety measure.

Reply:
in theory it should be impossible so i see no reason to leave it you can test it if you want